tests/test_access_route.py

```diff
@@ -42,9 +43,9 @@ def test_malformed_rfc_forwarded(self):
             headers={
                 'Forwarded': 'for'
             }))
-        self.assertEqual(req.access_route, ['127.0.0.1'])
+        self.assertEqual(req.access_route, [])
```

Comment:
Since the header can be malformed from client, we'd better add a document note for this.